diff --git a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
index 3cacb9a..44adda3 100644
--- a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
+++ b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
@@ -475,7 +475,6 @@
       var node = getNodeWithoutReactNamespace(calleeNode);
       if ("Identifier" !== node.type) return -1;
       switch (node.name) {
-        case "useEffect":
         case "useLayoutEffect":
         case "useCallback":
         case "useMemo":
diff --git a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.production.js b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.production.js
index 23e2b19..c0177a9 100644
--- a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.production.js
+++ b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.production.js
@@ -448,7 +448,6 @@ function getReactiveHookCallbackIndex(calleeNode, options) {
   var node = getNodeWithoutReactNamespace(calleeNode);
   if ("Identifier" !== node.type) return -1;
   switch (node.name) {
-    case "useEffect":
     case "useLayoutEffect":
     case "useCallback":
     case "useMemo":
