diff --git a/node_modules/react-native-video/lib/Video.web.js b/node_modules/react-native-video/lib/Video.web.js
index 85fc7ed..fbb4b6f 100644
--- a/node_modules/react-native-video/lib/Video.web.js
+++ b/node_modules/react-native-video/lib/Video.web.js
@@ -241,7 +241,6 @@ const Video = (0, react_1.forwardRef)(({ source, paused, muted, volume, rate, re
         }, onEnded: onEnd, style: videoStyle }));
 });
 const videoStyle = {
-    position: 'absolute',
     inset: 0,
     objectFit: 'contain',
     width: '100%',
diff --git a/node_modules/react-native-video/src/Video.web.tsx b/node_modules/react-native-video/src/Video.web.tsx
index 85f71a9..9b3957a 100644
--- a/node_modules/react-native-video/src/Video.web.tsx
+++ b/node_modules/react-native-video/src/Video.web.tsx
@@ -323,7 +323,6 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
 );
 
 const videoStyle = {
-  position: 'absolute',
   inset: 0,
   objectFit: 'contain',
   width: '100%',
